src/Raven.Server/ServerWide/Commands/IncrementClusterIdentitiesBatchCommand.cs

```diff
@@ -54,9 +54,21 @@ public override void FillJson(DynamicJsonValue json)
             json[nameof(Identities)] = new DynamicJsonArray(Identities);
         }
 
-        protected override BlittableJsonReaderObject GetUpdatedValue(long index, DatabaseRecord record, JsonOperationContext context, BlittableJsonReaderObject existingValue, RachisState state)
+        protected override BlittableJsonReaderObject GetUpdatedValue(long index, DatabaseRecord record, JsonOperationContext context,
+            BlittableJsonReaderObject existingValue, RachisState state)
         {
             throw new NotImplementedException();
         }
+
+        public override object FromRemote(object remoteResult)
+        {
+            var rc = new List<long>();
+            var obj = remoteResult as BlittableJsonReaderArray;
```

Comment:
Will get NRE here if the value isn't as expected. Use standard case to make sure we get good errors.